src/man/pam_sss.8.xml

```diff
@@ -249,6 +252,20 @@ auth sufficient pam_sss.so allow_missing_name
                     </para>
                 </listitem>
             </varlistentry>
+            <varlistentry>
+                <term>
+                    <option>allow_chauthtok_by_root</option>
+                </term>
+                <listitem>
+                    <para>
+                        By default the chauthtok PAM action will short-circuit to
+                        returning PAM_SUCCESS when pam_sss.so is invoked by root
+                        user.
+                        This option disables this behavior allowing to change
+                        auth tokens when running as root.
```

Comment:
If the intention here is to create a new paragraph, then it should be enclosed in standalone `<para>`.